### DISCUSSION TITLE
Does nni support one experiment to run on multiple jobs?

### DISCUSSION THREAD

**Discussion by caozixuan** [microsoft/nni#4065] [Q&A]
In our actual use, we need to use cluster management software such as slurm to apply for computing resources.  The resources that one job can apply for are limited. 
I wonder if nni supports to run multiple jobs to solve an experiment. For example, if I run `nnictl create --config nni/examples/trials/mnist-pytorch/config.yml`, can I start another job and run the experiment I just created to improve trialConcurrency?

**Comment by acured** [comment]
You want to manage trialConcurrency outside NNI?

**Reply by caozixuan** [reply]
yes, the resources I can apply for one job is limited.

**Reply by acured** [reply]
I am afraid that there is no way to change it dynamically.
